requirements.txt

```diff
@@ -1,4 +1,4 @@
-beaker-py
+beaker-py==1.35.0
```

Comment:
`beaker-py` version 2 breaks the functions we have here to get wandb/beaker IDs.